clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureNetworkClient.groovy

```diff
@@ -67,6 +78,40 @@ class AzureNetworkClient extends AzureBaseClient {
     result
   }
 
+  /**
+   * Retrieve an Azure load balancer for a give set of credentials, resource group and name
+   * @param creds the credentials to use when communicating to the Azure subscription(s)
+   * @param resourceGroupName the name of the resource group to look into
+   * @param loadBalancerName the of the load balancer
+   * @return a description object which represent a Load Balancer in Azure or null if Load Balancer could not be retrieved
+   */
+  AzureLoadBalancerDescription getLoadBalancer(AzureCredentials creds, String resourceGroupName, String loadBalancerName) {
+    try {
+      def currentTime = System.currentTimeMillis()
+      def item = this.getNetworkResourceProviderClient(creds).getLoadBalancersOperations().get(resourceGroupName, loadBalancerName).loadBalancer
+      if (item) {
+        def lbItem = getDescriptionForLoadBalancer(item)
+        lbItem.appName = AzureUtilities.getAppNameFromResourceId(item.id)
+        lbItem.tags = item.tags
+        lbItem.dnsName = getDnsNameForLoadBalancer(creds, AzureUtilities.getResourceGroupNameFromResourceId(item.id), item.name)
+
+        // TODO: investigate and add code to handle changes to publicIP resource associate with current load balancer
+        // There's a small probability that the publicIP resources associated with the current load balancer has changed
+        //  from the time we read the load balancer current properties at the beginning of the current closure/loop
+        //  and we should reflect that in the lastReadTime property.
+        // We currently don't use any of the publicIp properties other than the DNS so we don't need to address that now
+
+        lbItem.lastReadTime = currentTime
+        return lbItem
+      }
+    } catch (ServiceException e) {
+      log.info("getLoadBalancer(${resourceGroupName},${loadBalancerName}) -> Service Exception " + e.toString())
+      return null
+    }
+
+    null
```

Comment:
I don't know what you're talking about. I'm referring to the return null statement since both branches of the try/catch block have return statements this line would be unreachable.